src/com/google/javascript/jscomp/DefaultPassConfig.java

```diff
@@ -174,6 +174,17 @@ void maybeInitializePreprocessorSymbolTable(AbstractCompiler compiler) {
       }
     }
   }
+  
+  @Override
+  protected List<PassFactory> getAlwaysPreChecks() {
+    List<PassFactory> preChecks = new ArrayList<>();
+    
+    if (options.wrapGoogModulesForWhitespaceOnly) {
```

Comment:
In the debugger WHITESPACE_ONLY is called SKIP_NON_TRANSPILATION_PASSES.

Good point about not having closurePass. Maybe for now we should just check that we're in whitespace-only mode in addition to the compiler option, but eventually I'm ok with making it default to on (since using goog.modules in whitespace only mode doesn't really make sense without this option).